hack/lib/update-codegen.sh

```diff
@@ -196,6 +196,7 @@ crd-ref-docs \
 
 # Generate CRD YAML
 (cd apis &&
+    go install sigs.k8s.io/controller-tools/cmd/controller-gen@latest &&
```

Comment:
Could we do this when we generate the container image instead of here? Doing it here will cause it to download and compile this project every time CI runs this script, won't it? CI does that a lot in the `verify-codegen` jobs.

Reply:
Good point. This was done previous to updating the CI, so it should be removed here.